Algorithms/Sorting/bubble_sort.py

```diff
@@ -0,0 +1,24 @@
+# Name : Nabanita Dash
+# Email ID : dashnabanita@gmail.com
+
+# find the element in array by insertion sort
+# Code
+def bubble_sort(arr):
+    n = len(arr)
+    for i in range(n):
+        for j in range(0, n-i-1):
+            if arr[j] > arr[j+1] :
+                arr[j], arr[j+1] = arr[j+1], arr[j]
+
+# Test-Case
+arr = [1, 13, 78, 6, 45689 78, 678, 45689]
```

Comment:
Take the array as input from the user.

Reply:
I have taken ``arr`` as an input from user. I guess you are looking into ``#Test-Case`` rather than ``#Code``. I am including ``print function`` inside the  main function.

Reply:
I cannot find any input statement in this program.